### PR TITLE
[ZEVMIDST-146] Add notifications service

### PR DESCRIPTION
**Pull request by kacpersmal**

### CODE REVIEW

File: src/services/functions/notifications/DfT.ZEV.Services.Notifications.Handler/Function.cs [hunk 1]
```diff
@@ -45,6 +45,6 @@ public async Task HandleAsync(HttpContext context)
 
         var notification = JsonSerializer.Deserialize<Notification>(json, jsonSerializerOptions);
         _logger.LogDebug("Invoking service layer");
-        var result = _notificationsService.SendNotification(notification);
+        //var result = _notificationsService.SendNotification(notification);
```
Comment (cruddasj) [inline src/services/functions/notifications/DfT.ZEV.Services.Notifications.Handler/Function.cs:48]: Is this meant to have been commented out?

Comment (kacpersmal) [reply src/services/functions/notifications/DfT.ZEV.Services.Notifications.Handler/Function.cs:48]: It was not :D

File: src/common/DfT.ZEV.Common/Configuration/GoogleCloudConfiguration.cs [hunk 2]
```diff
@@ -14,12 +14,28 @@ public class TokenConfiguration
 
 }
 
+public class NotificationQueue
```
Comment (cruddasj) [inline src/common/DfT.ZEV.Common/Configuration/GoogleCloudConfiguration.cs:17]: Minor - but again - can we avoid created nested classes like this?

Comment (kacpersmal) [reply src/common/DfT.ZEV.Common/Configuration/GoogleCloudConfiguration.cs:17]: Allrighty then, im gonna split them

**Review by cruddasj — APPROVED** [review]

**Pull request merged**